### PR TITLE
Create workspace

### PR DESCRIPTION
**Pull request by markstur**

### CODE REVIEW

File: ?
Comment (stevemar) [inline ?:1]: If there are no workspaces will this result in a keyerror?

Comment (stevemar) [inline ?:1]: Awesome!

Comment (markstur) [reply ?:1]: Nope. It'll just be an empty list.

**Review by scottdangelo — APPROVED** [review]
LGTM and tested with the happy path. I didn't test with various error scenarios, i.e. missing ENV stuffs, nor with dodgy entries for WORKSPACE_ID. I assume Mark has tested...

**Review by stevemar — APPROVED** [review]

**Pull request merged**